server/src/Domain/Domain.csproj

```diff
@@ -7,6 +7,11 @@
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="FluentResults" Version="3.15.2" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="7.0.12" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="7.0.12">
```

Comment:
Not sure why added this?

Reply:
It wasn't my addition, I think it might be from the main merge and i wasn't sure if I could delete it